lib/sablon/image.rb

```diff
@@ -0,0 +1,17 @@
+module Sablon
+  class Image
+    include Singleton
+    attr_reader :definitions
+
+    Definition = Struct.new(:name, :data, :rid) do
+      def inspect
+        "#<Image #{name}:#{data}:#{rid}"
```

Comment:
Can we remove `data` here? Printing binary data in the console caused it freeze.